### PR TITLE
Included LDAP in php composer - removed un-needed rm line

### PR DESCRIPTION
**Pull request by homerr**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-kimai/blob/main/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

Added the LDAP requirement as specified in kimani docs

Removed un-needed `rm -rf /app/www/var` which broke adding additional composer packages 

## Benefits of this PR and context:

Closes issue #8 

## How Has This Been Tested?

Created new container from current latest image, changed to updated version no errors. Cannot test ldap

### CODE REVIEW

**Comment by homerr** [comment]
@nemchik could I ask for your review please mate

File: Dockerfile [hunk 1]
```diff
@@ -37,9 +37,10 @@ RUN \
   tar xf \
     /tmp/kimai.tar.gz -C \
     /app/www --strip-components=1 && \
-  rm -rf /app/www/var && \
```
Comment (nemchik) [inline Dockerfile:40]: note for self: i was wondering why remove this but i see we're doing it during the init as well https://github.com/linuxserver/docker-kimai/blob/88b5c29d964cbe80e5a710e5cde171fb24ad8c23/root/etc/s6-overlay/s6-rc.d/init-kimai-config/run#L29 to make room for symlinks to `/config`

File: Dockerfile [hunk 2]
```diff
@@ -37,9 +37,10 @@ RUN \
...
   echo "**** install composer dependencies ****" && \
   COMPOSER_MEMORY_LIMIT=-1 php -d memory_limit=-1 /usr/bin/composer install -d /app/www/ --optimize-autoloader --no-interaction && \
+  COMPOSER_MEMORY_LIMIT=-1 php -d memory_limit=-1 /usr/bin/composer require -d /app/www/ laminas/laminas-ldap && \
```
Comment (nemchik) [inline Dockerfile:42]: note for self: package recommended at https://www.kimai.org/documentation/ldap.html

**Review by nemchik — APPROVED** [review]
lgtm!

**Pull request merged**